satellite/audit/reservoir.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"math/rand"
+	"storj.io/storj/pkg/pb"
+)
+
+// Reservoir holds a certain number of segments to reflect a random sample
+type Reservoir struct {
+	Segments []*pb.RemoteSegment
```

Comment:
We should be keeping segment paths (type `storj.Path`) instead of `pb.RemoteSegment`. When the audit service selects a segment path from the reservoir, it must look up its up-to-date pointer from the metainfo db.